### PR TITLE
done

### PR DESCRIPTION
**Pull request by AliceGl**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,3 +1,17 @@
 import './style.css';
 
-console.log('Hello world');
+function tooltip_enable(event) {
+    var tooltip = document.createElement("span")
+    tooltip.innerHTML = "Hello World!";
+    tooltip.setAttribute("class", "tooltip");
```
Comment (coolassassin) [inline main.js]: DOM api has special methods to work with classes.
You can add new class like so:
```javascript
tooltip.classList.add('tooltip')
```

More info here:
https://developer.mozilla.org/en-US/docs/Web/API/Element/classList

File: main.js [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+    event.srcElement.appendChild(tooltip);
+}
+function tooltip_disable(event) {
+    event.srcElement.removeChild(event.srcElement.lastChild);
```
Comment (coolassassin) [inline main.js]: If you are trying to find a button with `event.srcElement`, you can get access easier by just `this`

File: index.html [hunk 3]
```diff
@@ -7,7 +7,10 @@
     <title>Vite App</title>
   </head>
   <body>
-    <div id="app"></div>
+    <div id="app">
+      <button id="on_hover">Tooltip on hover</button>
+      <button id="on_click">Tooltip on click</button>
```
Comment (coolassassin) [inline index.html:13]: Good job. You can also create a button with modal window 🙂

Comment (AliceGl) [reply index.html:13]: Oh, sorry, for me the demo looked like this and I didn't notice the third button 😅

**Pull request closed without merging**